core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsFiles.java

```diff
@@ -0,0 +1,37 @@
+package org.mobilitydata.gtfsvalidator.input;
```

Comment:
In general there are changes here an in 1504-... branch that touch the same files.
This will likely lead to conflicts.
I suggest to wait for 1504-... to be merged in master, then merge master back in this 1532-... branch and resolve the conflicts.